samples/rpc-commands.js

```diff
@@ -0,0 +1,28 @@
+const BtcTransactionHelper = require('../index').BtcTransactionHelper;
```

Comment:
Please add how to run this sample into the README.md file.

Reply:
Updated!